src/workers/parser/tests/__snapshots__/getOutOfScopeLocations.spec.js.snap

```diff
@@ -21,20 +22,26 @@ exports[`Parser.getOutOfScopeLocations should exclude non-enclosing function blo
 (35, 20) -> (37, 1)
 (39, 26) -> (41, 1)
 (43, 20) -> (45, 1)
+(47, 31) -> (49, 1)
+(51, 19) -> (62, 1)"
+`;
+
+exports[`Parser.getOutOfScopeLocations should not exclude in-scope inner locations 1`] = `
+"(1, 0) -> (1, 16)
+(3, 14) -> (27, 1)
+(29, 16) -> (33, 1)
+(35, 20) -> (37, 1)
+(39, 26) -> (41, 1)
+(43, 20) -> (45, 1)
```

Comment:
This might be a crazy idea, but i think it would be fun to write a helper function for our tests so that we can visualize what is in scope and what is out of scope.

for instance, we could write `expect(highlightInScopeLocations(..)).toMatchSnapshot()` and the locations in the code that are inscope are bolded with https://github.com/chalk/chalk

This would make it really fun to debug this code because you could quickly visualize what the algorithm outputs!

```
function parentFunc() {
  let MAX = 3;
  let nums = [0, 1, 2, 3];
  let x = 1;
  let y = nums.find(function(n) {
    return n == x;
  });
  function innerFunc(a) {
    return Math.max(a, MAX);
  }
  return innerFunc(y);
}
```

Reply:
@jasonLaster 

Do you mean printing out in-scope function in tests/fixtures/outOfScope.js like this?
``` 
// This parentFunc is in-scope so from here codes are in green
function parentFunc() {
  let MAX = 3;
  let nums = [0, 1, 2, 3];
  let x = 1;
  let y = nums.find(function(n) {
    return n == x;
  });
  function innerFunc(a) {
    return Math.max(a, MAX);
  }
  return innerFunc(y);
}
// to here codes are in green
```

Reply:
Yep, i might hack on this now that it has landed :)